### PR TITLE
updated issue templates for PrismLauncher

### PR DESCRIPTION
**Pull request by TayouVR**
fixes what https://github.com/PrismLauncher/PrismLauncher/issues/73 says in the brackets

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
 contact_links:
-  - name: PolyMC Matrix Support Room
-    url: https://matrix.to/#/#support:polymc.org
+  - name: Prism Launcher Matrix Support Room
+    url: https://matrix.to/#/#support:prismlauncher.org
```
Comment (flowln) [inline .github/ISSUE_TEMPLATE/config.yml]: do we have a matrix room tho? :thinking:

**Review by flowln — COMMENTED** [review]
don't forget the signoff! :o

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (TayouVR) [reply .github/ISSUE_TEMPLATE/config.yml]: no, but maybe we will, I guess i'll remove it for now

Comment (TayouVR) [reply .github/ISSUE_TEMPLATE/config.yml]: actually... do I just remove the whole section?
Or should I add the discord server instead maybe?

Comment (TayouVR) [reply .github/ISSUE_TEMPLATE/config.yml]: the discord server invite link I put in there should point to the Support channel.

**Review by flowln — APPROVED** [review]

**Comment by piprett** [comment]
Looks like duplicate of #47 ?

**Comment by TayouVR** [comment]
> Looks like duplicate of #47 ?

oh yep, you are right!

should I close?

**Comment by TheOPtimal** [comment]
yeah, looks like a duplicate

**Comment by Scrumplex** [comment]
funny that #47 and #74 do the same thing

**Review by AurorusFur — APPROVED** [review]

**Comment by TayouVR** [comment]
#47 is merged now, so I'm closing

I had also added the discord to the contact links, but I suppose that is irrelevant.

**Pull request closed without merging**